src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -5,51 +5,121 @@
+          <div v-if="key[1].constructor === Array"
+               :class="key[0] !== 'content' ? 'td content clickAble' : 'td content'">
+            <span v-for="(entry, newIndex) in key[1]"
+                  :key="newIndex"
+                  :class="index === currentDataShown ? 'singleEntry' : newIndex < arrayShownLimit ? 'singleEntry' : 'singleEntry hidden'"
+                  @click="key[0] !== 'content' ? searchFromAllValues(key[0], entry) : null">
+              {{ entry }} <br>
+            </span>
+            <button v-if="key[1].length > arrayShownLimit"
+                    :key="index + '-button' "
+                    class="attributeButton"
+                    @click="toggleShownData(index)">
+              {{ specificValueButtonText(index) }}
+            </button>
+          </div>
+          <div v-if="key[1].constructor !== Array" :class="key[0] !== 'content' ? 'td content clickAble' : 'td content'" @click="key[0] !== 'content' ? searchFromAllValues(key[0], key[1]) : null">
+            <span :class="key[0] === 'content' ? '' : 'singleEntry'"> {{ key[0] === 'content' ? displayContentValue(key[1]) : key[1] }}</span>
+          </div>
+          <button v-if="key[0] === 'content' && key[1].length > contentShownLength" class="contentButton" @click="allContentShownToggle()">
+            {{ contentButtonText }}
+          </button>
         </div>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-//import { mapState, mapActions } from 'vuex'
+import { mapState, mapActions } from 'vuex'
+import { requestService } from '../../services/RequestService'
 
 export default {
-  name: 'SearchSingleItemStandardInfo',
+  name: 'SearchSingleItemAllData',
   components: {  
   },
   props: {
-    result: {
-      type: Object,
+    id: {
+      type: String,
       required: true
     },
   },
   data () {
     return {
       allDataShown:false,
+      allData:{},
+      allContentShown:false,
+      currentDataShown:null,
+      contentShownLength:300,
+      arrayShownLimit:4
     }
   },
   computed: {
+    ...mapState({
+      results: state => state.Search.results,
+      results: state => state.Search.query,
+      searchAppliedFacets: state => state.Search.searchAppliedFacets,
+    }),
     allDataButtonText: function () {
-      return this.allDataShown ? 'Hide data ' : 'See data'
-    }
+      return this.allDataShown ? 'Hide raw data' : 'See raw data'
+    },
+    contentButtonText: function () {
+      return this.allContentShown ? 'Show less ↑' : 'Show all ↓'
+    },
   },
   mounted () {
   },
   methods: {
+    ...mapActions('Search', {
+      requestSearch: 'requestSearch',
+      requestFacets: 'requestFacets',
+      updateQuery: 'updateQuery',
+      updateSearchAppliedFacets:'updateSearchAppliedFacets',
+
+    }),
     toggleAllDataShown() {
       this.allDataShown = !this.allDataShown
+      if(Object.keys(this.allData).length === 0 && this.allData.constructor === Object) {
```

Comment:
Very defensive here with the Object constructor check - isn't _allData_  always === Object ?

Reply:
Good point, removed the redundency.